### DISCUSSION TITLE
Ad-Astra for 1.21.1 ?

### DISCUSSION THREAD

**Discussion by DKSteffenDE** [terrarium-earth/Ad-Astra#654] [General]
As the title implies, will there be a version for minecraft 1.21.1 ?

**Comment by DRD2k8** [comment]
I hope so. It's the best space mod out there.

**Comment by DeineLeber** [comment]
In the Discord they say AdAstra will be available for 1.21.1 but no Date announced :)

**Reply by DRD2k8** [reply]
Ad Astra hasn't been maintained for a year, I hope this mod isn't discontinued

**Comment by a1513** [comment]
NEOFROGE

**Comment by NoahGirling** [comment]
Wonder when it's coming! Been about 5 months

**Comment by Resp0und** [comment]
Hope for neoforge 1.21.1

**Comment by StealthyDrake12** [comment]
> As the title implies, will there be a version for minecraft 1.21.1 ?

(neoforge)

**Comment by Yukifuri-Yukari** [comment]
I hope too, cause i have to use it with my friend uses 1.21.1
lol

**Reply by Yukifuri-Yukari** [reply]
neo is already released 1.21.1, but ad isnt lol

**Reply by Yukifuri-Yukari** [reply]
it's the best space mod that i have ever seen

**Reply by Yukifuri-Yukari** [reply]
i must write a un-official version to play with my friend =(

**Comment by shotadft** [comment]
I would like to request 1.21.1 support from me too!

**Comment by DKSteffenDE** [comment]
it's been almost a year now, is there any news on a date for Ad-Astra for 1.21.1, neoforge?
or should I go with Stellaris instead, as that seems to be ready?

**Comment by JetVolcano** [comment]
3 More months until it's been a year since it's been updated

**Comment by ZR-official** [comment]
waiting for neoforge 1.21.8

**Comment by Yukifuri-Yukari** [comment]
hmm, or i'll write mod by myself if they didnt release new version before 2027. im learning how to write mods for minecraft.

**Comment by ShallIMove** [comment]
After the rewrite, I would like to not only rewrite the code and pOrt to 1.21.1, but also add a lot of new mechanics: radiation, pressure, atmospheric density, atmospheric composition (not the presence of oxygen or not, but the presence of suitable air in terms of atomic composition 78% N2, 21% O2, 1% argon, carbon dioxide , etc .There are a lot of things that can be added and a lot can be done from these mechanics to the players Using them to the fullest.
